modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/OrganizationChart.js

```diff
@@ -3,57 +3,134 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
+import ClayAlert from '@clayui/alert';
 import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
 import classnames from 'classnames';
-import {sub} from 'frontend-js-web';
 import PropTypes from 'prop-types';
-import React, {useEffect, useLayoutEffect, useRef, useState} from 'react';
+import React, {
+	useCallback,
+	useEffect,
+	useLayoutEffect,
+	useRef,
+	useState,
+} from 'react';
 
 import ChartContext from './ChartContext';
 import D3OrganizationChart from './D3OrganizationChart';
 import ManagementBar from './ManagementBar/ManagementBar';
+import ResultsBar from './ManagementBar/ResultsBar';
 import {getOrganization, getOrganizations} from './data/organizations';
 import MenuProvider from './menu/MenuProvider';
 import ModalProvider from './modals/ModalProvider';
 import InfoPanelProvider from './panels/InfoPanelProvider';
 import {VIEWS} from './utils/constants';
+import {findRoot} from './utils/findRoot';
 
 import '../style/main.scss';
 
+const MAX_DISPLAYED_ORGANIZATIONS = 10;
+const DEFAULT_PAGE_SIZE = 50;
```

Comment:
Can we put these in the constants file? I think the same value it's used also in RootOrganizationFilter.tsx

Reply:
In reality in RootOrganizationFilter 10 is the max value to freeze the selection, but I can move also that one constant to constants.ts